public/v4/apps/chatwoot-botpress-bridge.yml

```diff
@@ -0,0 +1,47 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: douglara/chatwoot-botpress-bridge:$$cap_version
```

Comment:
I just realized that this image is only pulled 53 times... We only typically merge popular repos (thousands of stars and tens of thousands of pulls).

But since you've already made a few changes here, I'll go ahead and add this for now.

Reply:
Yes I understand, I maintender this project and users suggested this ease to use =)
Thanks!